### DISCUSSION TITLE
Safari extension

### DISCUSSION THREAD

**Discussion by ilkin** [dailydotdev/daily#961] [Feature Requests]
Why don't you have safari extension?

**Comment by idoshamun** [comment]
Hi @ilkin,

We're trying to port our extension to Safari, but some technical challenges exist.
Hopefully, we will have news in the upcoming weeks.

**Comment by mofukommit** [comment]
@idoshamun still technical challenges? 
Safari extension would be great for devs working in Apple's ecosystem.

**Reply by idoshamun** [reply]
@mofukommit yes, there's a limitation with safari extensions and cookies last time we checked. This doesn't allow our authentication system to work properly

**Reply by mofukommit** [reply]
Thank you for the update!

**Comment by tiagomaricate** [comment]
As the hope never dies... some news about the safari limitations?

**Reply by idoshamun** [reply]
Unfortunately no 😢
